src/renderer/components/Login.tsx

```diff
@@ -215,14 +256,24 @@ export default class Login extends React.Component {
             password={mail_pw}
             onChange={this.handleCredentialsChange}
           />
-          <DeltaText>{tx('login_no_servers_hint')}</DeltaText>
-          <AdvancedButton onClick={this.handleUISwitchStateProperty.bind(this, 'showAdvanced')} id={'show-advanced-button'}>
-            {(showAdvanced ? <AdvancedButtonIconClosed /> : <AdvancedButtonIconOpen />)}
+
+          { provider_info?.before_login_hint &&
+          <div className={`before-login-hint ${(provider_info.status === C.DC_PROVIDER_STATUS_BROKEN && 'broken')}`}>
+            <p>
+              {provider_info.before_login_hint}
+            </p>
+            <ClickableLink href={provider_info.overview_page}>More Information at {provider_info.overview_page}</ClickableLink>
```

Comment:
I would prefer to use just "More info" as link text, like the android app does. Showing URLs is ugly, in my eyes. And we would have a consistent behaviour with the android app.

If you reject my suggestion: "information" must not be capitalized.

Reply:
if the decision goes towards "More info", android uses the string originally introduced by desktop, `more_info_desktop`

Reply:
IDK my hope with this was to don't force opening the link in the system browser. (privacy stuff) but we can also do the `More info` text, but maybe add an `(online)` to make clear that it goes to the browser?
Either way is fine for me, just wanted to throw that thought out here.

Reply:
What does "privacy stuff" mean concretely?

People who care for privacy probably have a system browser with decent privacy settings, I would assume. And the link doesn't reveal a lot: only that this person might be using Delta Chat, and has an email address with provider X. Both of which is probably already known to the system browser, or the connectivity provider.

Reply:
Any more opinions or reactions?

Reply:
We should merge it now(after requiring a new dc-node version) and do a separate pr to change it later to prevent this being stale.

Reply:
Why not discuss this as long as this is waiting anyway?

You wrote:
> we can also do the More info text

If you are OK with it, and no one objected my suggestion, and you don't fancy further discussion — why not just do it?